util/util.go

```diff
@@ -115,7 +115,7 @@ func ResolveName(name string, firstRegistry string, sc *types.SystemContext, sto
 	}
 	for _, registry := range searchRegistries {
 		if !registry.Blocked {
-			registries = append(registries, registry.URL)
+			registries = append(registries, registry.Location)
```

Comment:
This should be `registry.Prefix` actually, so that later lookups of the result of using this registry find and use the same registry entry.

Reply:
Alright, I changed it.